src/log/log.c

```diff
@@ -462,6 +462,10 @@ __log_file_header(
 		WT_ERR(__log_acquire(session, logrec->len, &tmp));
 	}
 	WT_ERR(__log_fill(session, &myslot, 1, buf, NULL));
+	/*
+	 * Make sure the header gets to disk.
+	 */
+	WT_ERR(__wt_fsync(session, tmp.slot_fh));
```

Comment:
Hi @sueloverso, thanks!  This generally looks good to me.

My one question is here: there is a comment about "writing into the main log" -- is that still valid?  If so, is there still a window where the main log has no header?